components/02-elements/form/input/Input.vue

```diff
@@ -47,11 +54,15 @@ export default {
       type: String,
       required: true
     },
+    tag: {
```

Comment:
Since we have just two possible options and one of them is default, IMO will be better to go with Boolean prop `textarea` instead this one.